pkg/version/version.go

```diff
@@ -3,6 +3,4 @@
 
 package version
 
-const (
-	Version = "0.34.0"
-)
+var Version = "develop"
```

Comment:
Question to understand: what does setting this to non-semver do for us?

Reply:
This was primarily to make it consistent with feedback received for imgpkg.

I am not sure it does technically much for us. It acts more as a placeholder that the version comes from 'somewhere else'. And that somewhere else is our build tooling (via go's LDD build flags).